source/messaging/organizing-channels.rst

```diff
@@ -1,9 +1,7 @@
-Organizing Mattermost
-=====================
+Organizing Channels
+====================
 
-The importance of organizing channels increases as Mattermost spreads to thousands of users.
-
-This document shares best practices from various Mattermost deployments including:
+The importance of organizing channels increases as Channels spreads to thousands of users. This document shares best practices from various Mattermost deployments including:
```

Comment:
```suggestion
The importance of organizing your conversations increases as Channels spreads to thousands of users. This document shares best practices from various Mattermost deployments including:
```

Reply:
My reasoning behind this is to clarify what's being organized and avoid redundancy.